src/main/java/org/example/ConfigFileReader.java

```diff
@@ -0,0 +1,33 @@
+package org.example;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ConfigFileReader {
+
+    private static final String PROPERTY_FILE_PATH = "src/main/resources/app.properties";
+    private Properties properties;
```

Comment:
```suggestion
    private static final String PROPERTY_FILE_PATH = "src/main/resources/app.properties";
    
    private Properties properties;
```
split constatns from class fields